DigitalLearningSolutions.Data.Tests/TestHelpers/UserTestHelper.cs

```diff
@@ -216,24 +216,51 @@ FROM Candidates
 
         public static void GivenDelegateUserIsInDatabase(DelegateUser user, SqlConnection sqlConnection)
         {
-            sqlConnection.Execute(
-                @"INSERT INTO Candidates (Active, CentreId, LastName, DateRegistered, CandidateNumber, JobGroupID,
-                    Approved, ExternalReg, SelfReg, SkipPW, PublicSkypeLink)
-                  VALUES (@Active, @CentreId, @LastName, @DateRegistered, @CandidateNumber, @JobGroupID,
-                    @Approved, @ExternalReg, @SelfReg, @SkipPW, @PublicSkypeLink);",
+            var userId = sqlConnection.QuerySingle<int>(
+                @"INSERT INTO Users
+                    (
+                        FirstName,
+                        LastName,
+                        PrimaryEmail,
+                        PasswordHash,
+                        Active,
+                        JobGroupID
+                    )
+                    OUTPUT Inserted.ID
+                    VALUES (@FirstName, @LastName, @EmailAddress, @Password, @Active, @JobGroupId)",
                 new
                 {
+                    user.FirstName,
+                    user.LastName,
+                    user.EmailAddress,
+                    user.Password,
                     user.Active,
+                    user.JobGroupId,
+                }
+            );
+            // TODO: Remove LastName_deprecated from this query once the not-null constraint is lifted

```

Comment:
I think it would be good to mark this TODO comment as part of the UAR journey. Don't know if we have a consistent set of rules for this seeing as we don't have clear tickets, but just staring it off as "TODO: UAR - " might be beneficial so we can easily search through the codebase for a set of undone UAR TODOs.

Reply:
done